app/containers/Header/styledComponents/logo.js

```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components';
```

Comment:
Do not create subfolder for styled components
file name needs to start from capital letter